client/__tests__/__mocks__/signup.mock.js

```diff
@@ -5,7 +5,7 @@ export const validSignup = [{
     request: {
         query: gql`
             mutation {
-                signup(email: "test@mail.com", password: "password1!", type: "resident", idNumber: "1")
+                signup(email: "test@mail.com", password: "password1!",confirmationPin:"11111", type: "resident", idNumber: "1")
```

Comment:
why is it hardcoded?

Reply:
We currently are not giving the residents a pin however should we choose to do so is setup in a way to make it possible.